src/core/utils/script-loader.ts

```diff
@@ -47,7 +48,7 @@ class ScriptLoader implements ScriptLoaderInterface {
 	loadScript(
 		src: string,
 		type = 'text/javascript',
-		isAsync = true,
+		isAsyncOrDeferOtherwise = true,
```

Comment:
I'm not sure, wouldnt be confusing to have more than one "is" under "boolean". Maybe it would be better (for readability) to change to some kind of dictionary-values e.g. with numbered values? 

maybe ... (its just a draft)
```
// somewhere higher
const WAIT_ATTTRIBUTE_TYPE = {
  DEFAULT: 1,
  ASYNC: 1,
  DEFER: 2,
};
```

and then... 
```suggestion
		isAsyncOrDeferOtherwise = WAIT_ATTTRIBUTE_TYPE.DEFAULT,
```

something like that?

Reply:
I would rather omit adding default value as if someone will forgot what it means it will slow down the whole processing and we don't want to add this option ;)